Cannot find name 'AbortSignal'

Hi,

When `import {CosmosClient} from '@azure/cosmos'`, the TS complains about missing type for AbortSignal. 

```
node_modules/@azure/cosmos/dist/index.d.ts:2414:19 - error TS2304: Cannot find name 'AbortSignal'.

2414     abortSignal?: AbortSignal;
```
Wonder if that's indeed missing.

--- a/consumer-test.js
+++ b/consumer-test.js
@@ -21,8 +21,10 @@ async function exec(cmd) {
     await exec("npm install --save ./..");
     console.log("Installing @azure/cosmos as a file dependency");
     for (const version of versions) {
-      console.log(`Compling with typescript@${version}`);
+      console.log(`Compling with typescript@${version} - Basic`);
       await exec(`npx -p typescript@${version} tsc ./test.ts --allowSyntheticDefaultImports true`);
+      console.log(`Compling with typescript@${version} - Custom lib`);
+      await exec(`npx -p typescript@${version} tsc ./test.ts --allowSyntheticDefaultImports true --lib es2018`);
     }
     process.exit(0);
   } catch (error) {

--- a/src/request/SharedOptions.ts
+++ b/src/request/SharedOptions.ts
@@ -1,3 +1,4 @@
+/// <reference lib="dom" />
 import { CosmosHeaders } from "../index";
 
 /**

